internal/rbd/replicationcontrollerserver.go

```diff
@@ -450,6 +455,19 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 		}
 	}
 
+	// when the images are demoted on both clusters and user requests for the
+	// resync of the image, the image mirror state will be unknown state in
+	// both clusters.
+	if state == upAndUnknown {
+		ready = true
+		for _, s := range mirrorStatus.PeerSites {
+			if imageMirroringState(s.State) != upAndUnknown {
+				util.UsefulLog(ctx, "peer site name=%s mirroring state=%s, description=%s and lastUpdate=%s", s.SiteName, s.State, s.Description, s.LastUpdate)
```

Comment:
@madhu-1, may be add a comma after peer site name?

Reply:
sure